src/main/java/com/gmail/Application.java

```diff
@@ -0,0 +1,17 @@
+package com.gmail;
+
+import com.gmail.entity.Car;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class Application {
+    private static final String APPLICATION_CONTEXT = "ApplicationContext.xml";
+
+    public static void main(String[] args) {
+        ApplicationContext context = new ClassPathXmlApplicationContext(APPLICATION_CONTEXT);
+        Car bmwSingleton = (Car) context.getBean("bmwSingleton");
```

Comment:
Please use the overload method. 
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/BeanFactory.html#getBean-java.lang.Class-
![image](https://user-images.githubusercontent.com/20267148/56869725-c643e180-6a0c-11e9-84dc-8fa39f95df2d.png)